src/js/components/about/MoreInfo.jsx

```diff
@@ -25,7 +25,7 @@ const MoreInfo = () => (
                 <a
                     target="_blank"
```

Comment:
This link doesn't need to open in a new tab since the new link is part of usaspending